demosplan/DemosPlanCoreBundle/ResourceTypes/ProcedureMapSettingResourceType.php

```diff
@@ -131,14 +158,12 @@ protected function getProperties(): ResourceConfigBuilderInterface
                 ->readable(false, fn (ProcedureSettings $procedureSettings): ?array => $this->convertJsonToCoordinates($procedureSettings->getTerritory()));
         }
 
-        if ($this->currentUser->hasPermission('field_master_procedure_default_bounding_box')) {
-            $configBuilder->defaultBoundingBox
-                ->readable(false, function (ProcedureSettings $procedureSetting): ?array {
-                    $masterTemplateMapSetting = $this->masterTemplateService->getMasterTemplate()->getSettings();
+        $configBuilder->defaultBoundingBox
```

Comment:
Permission `field_master_procedure_default_bounding_box` is not needed any more?

Reply:
exactly, it is not needed anymore

Reply:
tbh I kep the permission still on the other PRs per project, just in case FE says something different on monday